kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -0,0 +1,84 @@
+package com.kodilla.testing.shape;
+
+
+import org.junit.jupiter.api.*;
+
+@DisplayName("TDD: ShapeCollector Test Suite")
+public class ShapeCollectorTestSuite {
+
+    private static int testCounter = 0;
+
+    @BeforeAll
+    public static void beforeAllTests() {
+        System.out.println("Start zestawu testow...");
+    }
+
+    @AfterAll
+    public static void afterAllTests() {
+        System.out.println("Testy zostaly zakonczone");
+    }
+
+    @BeforeEach
+    public void beforeEveryTest() {
+        testCounter++;
+        System.out.println("Przeprowadzanie testu nr #" + testCounter);
+    }
+
+    @Test
+    void testAddFigure(){
+        //Given
+        ShapeCollector shapeCollector = new ShapeCollector(new Triangle("Trójkąt", 3, 4));
+
+        //When
+        shapeCollector.addFigure(new Triangle("Trójkąt", 3, 4));
+
+        //Then
+        Assertions.assertEquals(1, shapes.size());
```

Comment:
To sprawdza, czy jest jedna figura - lepiej sprawdzic, czy to jest rzeczywiscie ta sama figura, czyli trojkat o podstawie 3 i wysokosci 4